python/test/unit/fem/test_fem_pipeline.py

```diff
@@ -328,22 +128,25 @@ def test_vector_P_tp(family, degree, cell_type, datadir):
     run_vector_test(mesh, V, degree)
 
 
-# TODO: Implement DPC spaces
 @parametrize_cell_types_quad
-@pytest.mark.parametrize("family", ["DQ"])
-# @pytest.mark.parametrize("family", ["DQ", "DPC"])
+@pytest.mark.parametrize("family", ["DQ", "DPC"])
 @pytest.mark.parametrize("degree", [1, 2, 3])
 def test_dP_quad(family, degree, cell_type, datadir):
+    if family == "DPC":
+        pytest.skip()  # These space currently not implemented in basix
```

Comment:
pytest.skip() takes a string, do not put this into a comment. And below as well.